Fix #5628 - Remove unwanted test case

smrajagopal opened the pull request:
Removed test case requiring only 5 'return' statements
be used in 'Waypoint: Chaining If Else Statements'

`npm run test-challenges` returned `Pass` also tested locally.

![image](https://cloud.githubusercontent.com/assets/1938039/12068859/6c0288e8-b03d-11e5-9d4a-1f14bdfcf7df.png)

smrajagopal commented:
Well, I will close this PR. Because two of us worked at the same time and this PR was created a minute later one another one which should be merged.

ltegman commented:
Please put `closes #5628` in the body of your PR so that the issue will be autoclosed when this is accepted. Also, per @SaintPeter's request in that thread could you just change this test to check for > 5 rather than remove it? After you make this change you can `git add . && git commit --amend && git push -f` to amend this commit and force push the change to this PR. Thanks! :smile:

smrajagopal commented:
@ltegman 
I accidentally closed this one instead of closing the other [duplicate PR](https://github.com/FreeCodeCamp/FreeCodeCamp/pull/5671)

This one is all good to proceed with review by FCC Collaborator.

smrajagopal committed:
Fix #5628 - Update test case to fix logic around return statements

Fixed test case to allow 5 or more 'return' statements
to be used in 'Waypoint: Chaining If Else Statements'

smrajagopal commented:
@ltegman 
Fixed in 63f324c147a08ccfbc69e5f8b5647dfcaecfcd65
closes #5628

ltegman commented:
You have to edit the main message of your PR for it to close the issue, but I'll just close it myself this time. This PR looks good now, thanks!

gayfordjames commented:
Started today
